pkg/scheduler/internal/queue/scheduling_queue.go

```diff
@@ -739,106 +739,90 @@ func (p *PriorityQueue) AssignedPodUpdated(pod *v1.Pod) {
 	p.lock.Unlock()
 }
 
+// MoveAllToActiveOrBackoffQueue moves all pods from unschedulablePods to activeQ or backoffQ.
+// This function adds all pods and then signals the condition variable to ensure that
+// if Pop() is waiting for an item, it receives the signal after all the pods are in the
+// queue and the head is the highest priority pod.
+func (p *PriorityQueue) MoveAllToActiveOrBackoffQueue(event framework.ClusterEvent, preCheck PreEnqueueCheck) {
```

Comment:
nit: would you mind moving it to original place, so the PR's changeset would be review-friendly.

Reply:
Okay, done. Just beware that the order of the functions is then probably not how one would normally present them when starting from scratch.